packages/guides-restructured-text/src/RestructuredText/Utility/AnnotationUtility.php

```diff
@@ -0,0 +1,42 @@
+<?php
+
+declare(strict_types=1);
+
+namespace phpDocumentor\Guides\RestructuredText\Utility;
```

Comment:
Abstract classes kind of cry to be extended? Wouldn't it make more sense to make it final? Not sure about static function, they are hard to test and it might be a good idea to also cover these functions with tests?

Reply:
switched to static functions but made the class final